src/main/java/task/Deadline.java

```diff
@@ -0,0 +1,25 @@
+package task;
+
+import task.Task;
+
+public class Deadline extends Task {
+    private String time;
+
+    public Deadline (String desc, String time, Boolean status) {
```

Comment:
Consider standardizing your use of boolean type to use either the Boolean object or boolean primitive so that your code is consistent. While `Boolean` and `boolean` essentially work the same (`Boolean` object provides more functionality under the hood), it is recommended that you stick to either one of them for consistency purposes.